client/src/models/imageScrubber.js

```diff
@@ -48,6 +48,19 @@ const imageScrubber = {
         return { ...state, byId }
       }
     },
+    receiveStatus(state, payload) {
+      /**
+       NOTE: May need to get access to tree store to get the tree to update in the TreeTable - 03/31/19
+       // state.trees.data.forEach(x => {
```

Comment:
Not sure if you have to implement this, but might be better with an `indexOf( x => x.id === payload.id)` and then just `state.tress.data[index].active = payload.active`

Reply:
not sure either, but if its updating the tree object in `imageScrubber` should the property `active` be updated on the same tree object tree state as well for consistency?

I'll push up the style fix and leave it commented, for the future.

Reply:
If it affects the UI (like there's a red box or something) then it should probably be updated for synchronicity's sake. On the other hand, since you hard code the values to either activate or deactivate, you don't have to worry about app state vs. database state for updating.

It's probably fine if you aren't going to show anything as different though